src/Classes/MyRadioException.php

```diff
@@ -94,9 +95,9 @@ public function uncaught()
             //Configuration is available, use this to decide what to do
             if (!$silent
                 && Config::$display_errors
-                || (class_exists('\MyRadio\MyRadio\CoreUtils')
+                || (class_exists('\MyRadio\MyRadio\AuthUtils')
```

Comment:
I'm not sure how class_exists behaves - does it check if it can be loaded via autoloader?
CoreUtils could be checked directly as it was needed for early in the setup process, although AuthUtils might be too.

Needs checking, I think.